vuepress/docs/next/tutorials/consume/mt-redis.md

```diff
@@ -28,16 +28,16 @@ For more information, see [Entando Multitenancy]()
 ```
 >Both REDIS_ACTIVE and REDIS_SESSION_ACTIVE need to be set to "true" to enable the storage of HTTP sessions. If only REDIS_ACTIVE is set to "true", Redis is used just for the cache.  
   
-4A. For a single node implementation, also add this variable : 
+4A. For a single node implementation, also add the variable and value for your Redis host address: 
 
 ```
-    - name: REDIS_ADDRESS
-      value: YOUR-REDIS-HOSTNAME
+  - name: REDIS_ADDRESS
+    value: YOUR-REDIS-HOSTNAME
 ```
-4B. For Redis in HA with a sentinel, insert the comma separated list of nodes like this example:
+4B. For Redis in high availability with Sentinel, insert the comma separated list of nodes like in this example:
 ```   
-    - name: REDIS_ADDRESSES
-      value: redis-node-0.redis-headless.test-mt-720.svc.cluster.local:26379,redis-node-1.redis-headless.test-mt-720.svc.cluster.local:26379,redis-node-2.redis-headless.test-mt-720.svc.cluster.local:26379
+  - name: REDIS_ADDRESSES
+    value: redis-node-0.redis-headless.test-mt-720.svc.cluster.local:26379,redis-node-1.redis-headless.test-mt-720.svc.cluster.local:26379,redis-node-2.redis-headless.test-mt-720.svc.cluster.local:26379
 ```
 
-5. Scale the entando-de-app deployment back up to 1 and check the system for any issues.
\ No newline at end of file
+5. Scale the `entando-de-app` deployment back up to 1 and check the system for any issues.
```

Comment:
Add note about [ENG-4634](https://entando.myjetbrains.com/youtrack/issue/ENG-4634) Redis HA with sentinel - REDIS_USE_SENTINEL_EVENTS (true by default) to trigger using master switching in case of failure